sciLifeLab_utils/run_QC_analysis.py

```diff
@@ -47,7 +47,14 @@ def main(args):
         flowcells_dirs  = [os.path.join(sample_data_dir,flowcell) \
                 for flowcell in os.listdir(sample_data_dir) \
                 if os.path.isdir(os.path.join(sample_data_dir,flowcell))]
-
+        
+        # to adapt the diretory structure of X runs
```

Comment:
@senthil10 you really beat me on time, I was planning to do this tomorrow :-D!!!! great

This applies to all projects that are organised on Irma, so not only HiSeqX. Can you change the comment?